GitSearch/Application/Search/SearchController.swift

```diff
@@ -14,7 +14,8 @@ class SearchController: UIViewController, UITextFieldDelegate {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-    
+        view.backgroundColor = UIColor.mainTint
```

Comment:
When I did this thing in child's init - I had a problem with blank black strip in top (instead of time and ect.)
So I put it here, but its breaking a little main idea.